Marlin/src/pins/pins.h

```diff
@@ -277,33 +277,33 @@
 //
 
 #elif MB(RAMPS_14_RE_ARM_EFB)
-  #include "pins_RAMPS_RE_ARM.h"      // LPC1768                                    env:LPC1768
```

Comment:
The LPC1768_debug_and_upload was removed

Reply:
I wasn't sure what to do about this, I followed the Due model of having two build versions one with and one without the traceback symbols included (as I think they use extra flash). So I added the LPC1768_debug_and_upload env back in. However I've messed up the edit of the ini file as I should have removed the option to include the traceback symbols from the "normal" LPC1768 env. Perhaps it makes sense to always include the symbols as we are not short of flash space?

Reply:
I think it will be best to comment them out by default so devs can enable them in necessary and keep the platformio ini file simpler with only 1 env

Reply:
Fine by me.